scripts/import_users.py

```diff
@@ -66,45 +72,108 @@ def print_total_weeks_by_user(df):
         print(result.to_string(index=False))
 
     def print_duplicate_hashes(df):
-        # df.pivot_table(index=["SHA256_hash"], aggfunc="size").sort_values()
         df["WeekEndingDates"] = df["WeekEndingDates"].apply(lambda x: [INDEX_TO_WEEKS[y] for y in x])
         dupe_hashes = df[df.duplicated("SHA256_hash", keep=False)]
         dupe_hashes_count = len(dupe_hashes)
         print(f"There are {dupe_hashes_count} duplicate hashes:")
         print(dupe_hashes)
         dupe_hashes.to_excel(DUPLICATE_HASHES_FILENAME)
 
+    # row2 will be prioritized if duplicate found, so make sure row2, and not row1, is the PUA row
+    def insert_sorted(row1, row2, first_hashes, is_both_ui):
```

Comment:
Could we name row2 'pauRow' instead?

Reply:
no, it gets called in two places, and in one scenario it's the other UI row (part time vs full time)... but i'll try to make that more obvious in the code or comments